containers/roles/foreman/tasks/install-puma.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 'Install packages'
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - rh-ruby22-ruby-devel
+    - gcc
+    - gcc-c++
+    - make
+    - automake
+    - kernel-devel
+    - openssl-devel
+    - libxslt-devel
+
+- name: 'Install puma-rails'
+  shell: scl enable tfm 'gem install puma-rails'
```

Comment:
You have a TODO somewhere to package this? It'd be nice if in the future foreman containers don't need a development stack installed to keep the size down.